blogposts/announcing-sourcegraph-3.6.md

```diff
@@ -0,0 +1,162 @@
+
+## 30% indexed search latency improvement for small and medium instances
+
+![chart showing 30% improvement of search latency](images/3.6-search-latency.png "30% improvement of search latency")
+
+We have made several changes to our search algorithm that improve performance and reduce peak memory usage. Our benchmarks show that the **99th percentile latency** of indexed searches in small to medium instances (less than 3000 repositories) **is reduced by up to 30%**. We will be continuing work to further benchmark and optimize Sourcegraph for our customers with 30,000+ repositories.
+
+## Add quick links to repositories or important resources below the search bar
+
+TO DO: VIDEO
+
+<!--
+<p class="container">
+  <div style="padding:56.25% 0 0 0;position:relative;">
+    <iframe src="https://player.vimeo.com/video/{ID}?color=0CB6F4&amp;title=0&amp;byline=" style="position:absolute;top:0;left:0;width:100%;height:100%;" frameborder="0" webkitallowfullscreen="" mozallowfullscreen="" allowfullscreen=""></iframe>
+  </div>
+  <p style="text-align: center"><a href="https://vimeo.com/{ID}" target="_blank">View on Vimeo</a></p>
+</p>
+-->
+
+[Quick links](https://docs.sourcegraph.com/user/quick_links) can now be added below the search bar on the home page and search results page. This makes it easier for teams to quickly access the Sourcegraph pages they care most about such as your team’s main repository, or have easy access to external resources like your CI system or team documentation.
+
+Add a quick link to global, organization, or user settings with the `quicklinks` field:
+
+```
+{
+  // ...
+  "quicklinks": [
+    {
+      "name": "Sourcegraph docs",
+      "url": "https://docs.sourcegraph.com"
+    },
+    {
+      "name": "Main repository",
+      "url": "/github.com/sourcegraph/sourcegraph"
+    }
+  ]
+  // ...
+}
+```
+
+## Added support for Bitbucket Cloud external service configuration
+
+TO DO: VIDEO
+
+<!--
+<p class="container">
+  <div style="padding:56.25% 0 0 0;position:relative;">
+    <iframe src="https://player.vimeo.com/video/{ID}?color=0CB6F4&amp;title=0&amp;byline=" style="position:absolute;top:0;left:0;width:100%;height:100%;" frameborder="0" webkitallowfullscreen="" mozallowfullscreen="" allowfullscreen=""></iframe>
+  </div>
+  <p style="text-align: center"><a href="https://vimeo.com/{ID}" target="_blank">View on Vimeo</a></p>
+</p>
+-->
+
+Previously, Bitbucket Cloud repositories had to be added and configured as individual git repositories. In Sourcegraph 3.6 we added first class support for connecting [Bitbucket Cloud repositories](https://docs.sourcegraph.com/admin/external_service/bitbucket_cloud). In this release, all repositories will be added for the username configured, and teams listed that the Bitbucket Cloud app password grants access to.
```

Comment:
```suggestion
Sourcegraph 3.6 now supports repository syncing for [Bitbucket Cloud](https://docs.sourcegraph.com/admin/external_service/bitbucket_cloud). In this initial release, all repositories accessible by the token for the user (including associated teams) will be synced. Support for excluding repositories is slated for 3.7.
```

@Unknwon, @christinaforney I'm presuming we are adding exclude rule support in a future release?

Reply:
Yes, but don't want to specify which release since 3.7 is such a short one for us with GopherCon + hack week.